README.md

```diff
@@ -8,18 +8,112 @@ It's a perfect match with Web5 DWNs since a PWA can work offline and DWN has a s
 
 Aside from that, the advanced PWA capabilities such as modifying fetch requests allow us to extract the most such as displaying images with DRLs in the traditional `src` attribute.
 
-## Introduction
+## Running Instructions
 
-Orient users to the project here. This is a good place to start with an assumption
-that the user knows very little - so start with the Big Picture and show how this
-project fits into it. It may be good to reference/link the broader architecture in the
-`collaboration` repo or the developer site here.
+### Running Development Environment Locally
 
-Then maybe a dive into what this project does.
+Requirements:
 
-Diagrams and other visuals are helpful here. Perhaps code snippets showing usage.
+- node v20.x (`nvm use 20`)
+- pnpm v9 (`npm i -g pnpm`)
+- docker (to run a local DWN server)
 
-Project leads should complete, alongside this `README`:
+```sh
+pnpm i
+docker compose up -d
+pnpm dev
+```
+
+### Building App
+
+```sh
+pnpm build
+```
+
+Deploy the `dist` folder to the server. It's just a static PWA! Please make sure all the settings are optimized for production, including your application descriptions, icons, service workers, polyfills and configurations (ie. check vite-pwa eslint recommendations below).
+
+## Application Structure
+
+- It's just a **Traditional React Application** with a PWA on top using `vite-pwa`
+  - `src/pages` - contains the page components
+  - `src/App.tsx` - contains the main App component
+  - `src/main.tsx` - contains the main entry point
+  - `src/sw.ts` - contains the service worker setup, including the new web5 `activatePolyfills` which will setup DRLs (and DWN store offline management in the future)
```

Comment:
Love this explanation! Really helped me. Suggestion: I think it might be helpful to just write out what DRL stands for the first time it's mentioned because I'm not familiar with this acronym and I think our community may wonder as well.

Reply:
Done!